test/test_pallas.py

```diff
@@ -654,9 +657,16 @@ def _test_ragged_paged_attention(
         head_dim,
         page_size,
         num_pages,
-        dtype,
+        q_dtype,
+        kv_dtype,
         max_num_batched_tokens=max_num_batched_tokens,
         max_num_seqs=max_num_seqs)
+    k_scale = 0.5 if kv_dtype in [torch.float8_e5m2] else None
+    v_scale = 0.5 if kv_dtype in [torch.float8_e5m2] else None
+    if num_heads[1] == 1 and kv_dtype in [torch.float8_e5m2]:
```

Comment:
nit: could you use a var num_kv_heads=num_heads[1] as it's not obvious

Also, could you remind me why attn kernel cannot support this case?